drivers/sensor/hx7xx/Kconfig

```diff
@@ -0,0 +1,48 @@
+# HX711 Weight Scale interface Sensor configuration options
+
+# Copyright (c) 2020, George Gkinis
+# SPDX-License-Identifier: Apache-2.0
+
+config HX7XX
+	bool "Common HX7xx driver for multiple Avia ADCs"
+
+config HX710
+	bool "HX710/HX712/HX720 Weight Scale Sensor"
+	default y
+	depends on HX7XX
```

Comment:
This seems a little odd for the user to have to set `CONFIG_HX7XX=y`. Why not `select HX7XX` and drop the prompt on the `config HX7XX`?